pyodide_http/_requests.py

```diff
@@ -30,7 +32,12 @@ def request(method, url, **kwargs):
         response.headers = CaseInsensitiveDict(resp.headers)
         # Set encoding.
         response.encoding = get_encoding_from_headers(response.headers)
-        response.raw = BytesIO(resp.body)
+        if issubclass(type(resp.body), IOBase):
```

Comment:
`isinstance(resp.body, IOBase)` would work here right? I would prefer that syntax

Reply:
yep